.pre-commit-config.yaml

```
@@ -1,19 +1,31 @@
--   repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: v2.5.0
-    hooks:
-    -   id: trailing-whitespace
-    -   id: flake8
-    -   id: check-yaml
-    -   id: check-added-large-files
-    -   id: debug-statements
-    -   id: end-of-file-fixer
-        exclude: '^.+?\.json$'
--   repo: git://github.com/asottile/reorder_python_imports
-    rev: v2.1.0
-    hooks:
-    -   id: reorder-python-imports
--   repo: git://github.com/pre-commit/mirrors-eslint
-    sha: v3.14.0
-    hooks:
-    -   id: eslint
-        additional_dependencies: ['eslint', 'eslint-plugin-html', 'eslint-config-airbnb', 'eslint-plugin-import', 'eslint-plugin-jsx-a11y']
+repos:
+    -   repo: https://github.com/python/black
+        rev: 21.12b0
+        hooks:
+            -   id: black
+                types_or: [ python, pyi ]
+                exclude: '^.+?\.template$'
+    -   repo: https://github.com/pycqa/flake8
+        rev: 4.0.1
+        hooks:
+            -   id: flake8
+                exclude: '^.+?\.template$'
+    -   repo: git://github.com/pre-commit/pre-commit-hooks
```

```suggestion
    -   repo: https://github.com/pre-commit/pre-commit-hooks
```

A recent change in github security means that the unauthenticated `git://` protocol URLs will no longer work. I am making similar changes in our other repos: https://github.com/learningequality/kolibri/pull/8987

---

changed